src/NuGet.Core/NuGet.Packaging/NuGetExtractionFileIO.cs

```diff
@@ -88,7 +88,7 @@ private static FileStream DotnetCoreCreateFile(string path)
         private static FileStream MonoPosixCreateFile(string path)
         {
             var fileStream = File.Create(path);
-            PosixChmod(path, _unixPermissions);
+            _ = PosixChmod(path, _unixPermissions);
```

Comment:
As per https://www.unix.com/man-page/posix/3p/chmod/, `Upon  successful  completion,  these  functions	shall  return 0.  Otherwise, these functions shall return -1 and set errno to indicate the error. If -1 is returned, no change to the file mode occurs.`. I am okay if you want to create a follow-up issue for this because there is no change to the existing functionality.

Reply:
I created follow up issue https://github.com/NuGet/Home/issues/11148